pkg/evm/evm.go

```diff
@@ -259,15 +261,80 @@ func SendTransaction(
 	return err
 }
 
+func FindOutScheme(rpcURL string) (ethclient.Client, string, error) {
```

Comment:
I feel like it will be more flexible if this detects and returns scheme not `ethclient.Client`
in this case it will have more generic use